website/docs/docs/dbt-cloud-apis/sl-python-sdk.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Python SDK"
+id: sl-python
+description: "Learn how to use the dbt Semantic Layer SDK for Python library to interact with the dbt Semantic Layer."
+tags: [Semantic Layer, APIs]
+keywords: [dbt Cloud, API, dbt Semantic Layer, python]
+sidebar_label: "Python SDK"
+---
+
+# Python SDK <Lifecycle status="beta"/>
```

Comment:
Do we need to mention anything about Python versions in this doc?

Reply:
good shout!